code/modules/plumbing/plumbers/bottler.dm

```diff
@@ -49,31 +51,30 @@
 ///changing input ammount with a window
 /obj/machinery/plumbing/bottler/interact(mob/user)
 	. = ..()
-	var/vol = min(100, round(input(user,"maximum is 100u","set ammount to fill with") as num|null, 1))
+	var/vol = clamp(round(input(user,"maximum is 100u","set ammount to fill with") as num|null, 1), 1, 100)
```

Comment:
get rid of vol var, just make wanted amount = the clamped stuff

i can't suggest this because you deleted lines